common/lib/capa/capa/responsetypes.py

```diff
@@ -833,9 +939,32 @@ def get_score(self, student_answers):
         """
         # log.debug('%s: student_answers=%s, correct_choices=%s' % (
         #   unicode(self), student_answers, self.correct_choices))
+        
+        tree = self.xml
+        problem_xml = tree.xpath('.')
+
+        # Partial credit type - can set 'points' only at the moment.
+        credit_type = problem_xml[0].get('partial_credit', default=False)
+
+        try:
+            credit_type = str(credit_type).lower().strip()
+        except ValueError:
+            _ = self.capa_system.i18n.ugettext
```

Comment:
Oh, the underscore. Sorry, I was confused as to which line was being commented on.

This is another one of those pieces that I copied from elsewhere. Honestly I have no idea why that's there, but the same pattern appears in literally 25 other places in the file.